mbed_lstools/platform_database.py

```diff
@@ -287,6 +287,10 @@
         u'RIOT': u'RIOT',
     },
     u'jlink': {
+        u'X729475D28G': {
+            u'platform_name': u'NRF51_DK',
+            u'jlink_device_name': u'nRF51422_xxAC'
```

Comment:
Shouldn't the `jlink_device_name` be `nRF51822_xxAC`?

Reply:
No, the DK actually has the 422 on it, which can run code written for a few of their chips.